protocol/avdl/chat1/notify.avdl

```diff
@@ -142,7 +142,6 @@ protocol NotifyChat {
   @lint("ignore")
   void ChatLeftConversation(keybase1.UID uid, ConversationID convID);
 
-  @notify("")
```

Comment:
what's the reason behind making a notification a call?

Reply:
I'm not sure what part of the stack is to blame here, but I found that if I had calls going out close enough together, they could get randomly out of order on the frontend. The result was the blue bar would get stuck since my "sync started" call would land after "sync finished". Note that in these cases the service logs would indicate the order was correct. I changed this to be a call to try and mitigate that, although I am fine with changing it back. I removed the "sync started" call that would cause this to happen anyway.